### ISSUE TITLE
Contributions to focalboard documentation

### ISSUE THREAD

**Issue by derrickmehaffy** [mattermost/focalboard#140]
**Summary:**
Apologies if this is not in the correct location, while searching for the focalboard docs I notice they are not located in this repo and didn't see a mattermost repo for them. If the documentation is also open-source I would love to assist in providing more up to date guides for deployment on updated OSes.

Currently the only server deployment documentation is for Ubuntu 18.04 which at this point is 3 years old and is no longer the LTS version of Ubuntu. 

**Steps to reproduce the behavior:**
N/A

**Expected behavior:**
N/A

**Screenshots:**
N/A

**Platform:**
N/A

**Additional context:**
If you have a link to the focalboard documentation repo I would happily submit a few PRs

**Comment by chenilim** [comment]
Hi @derrickmehaffy, the docs (http://www.focalboard.com website) are checked in under the [website folder](https://github.com/mattermost/focalboard/tree/main/website). This is currently manually deployed (by me), but you can edit and test changes to that folder locally (following the steps in the readme there).

Yes! Thanks for contributing any additional documentation, or clarifying the existing ones. Much appreciated!

**Comment by it33** [comment]
@chenilim would you be open to having someone from community migrate the docs out of the website folder and add them to this repo? 

It would make it easier to contribute to just a collection of markdown pages, rather than navigate through the website. 

We could have focalboard.com as just a shell pointing to `.md` files in this repo. 

We did this when Mattermost was in its earliest days and it added to the velocity of the project when all the docs were in one place, and really easy to read and update. 

Sometimes people want to contribute install guides and trouble shooting tips, etc. and having everything one repo is amazing for that. 

Over time we would migrate the docs out, but in the early days the energy is highest if we could have like a /docs folder somewhere with install guides, user guides, etc.

**Comment by chenilim** [comment]
Yeah @it33, that works! @derrickmehaffy and anyone else: feel free to submit PRs to add a /docs folder, and migrate the [Ubuntu download guide](https://github.com/mattermost/focalboard/blob/main/website/site/content/download/personal-edition/ubuntu.md) and [setup guide](https://github.com/mattermost/focalboard/blob/main/website/site/content/guide/server-setup/_index.md) (which probably should be combined) to start with.

We can point the website to those docs, and figure out the best place to put them later. There's a way to [deploy Hugo websites to GitHub pages](https://gohugo.io/hosting-and-deployment/hosting-on-github/) if someone wants to take a crack at it.

**Comment by it33** [comment]
Oh @chenilim sorry, I misunderstood, I thought the website was in a different project, I see it's now just in this repo. Whoops :) 

Okay, in that case, we need to make it easier to find stuff and update it. 

@derrickmehaffy I think you can find the markdown to generate https://www.focalboard.com/download/personal-edition/ubuntu/ in this repo at https://github.com/mattermost/focalboard/blob/main/website/site/content/download/personal-edition/ubuntu.md

@chenilim would we want to update the Ubuntu 18.04 instructions with instructions for the latest LTS, or add an additional page for it, if this is something @derrickmehaffy wants to work on?

**Comment by chenilim** [comment]
No worries. :) We'll eventually reorganize the docs anyway. Yes, adding to the existing Hugo pages is fine. Please add a separate page for the latest Ubuntu version (and / or other distros). I've run into cases myself where I want to install on 18.04 and the steps are slightly different. (And incidentally, I always find the [Digital Ocean docs](https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-18-04) the best, which have different pages per version).

We'll have to make a call as to what versions to officially support and test on later. I did a quick search, but couldn't find a "Ubuntu server market share by version" report. If anyone has a pointer or more insight, please share.

**Comment by derrickmehaffy** [comment]
> No worries. :) We'll eventually reorganize the docs anyway. Yes, adding to the existing Hugo pages is fine. Please add a separate page for the latest Ubuntu version (and / or other distros). I've run into cases myself where I want to install on 18.04 and the steps are slightly different. (And incidentally, I always find the [Digital Ocean docs](https://www.digitalocean.com/community/tutorials/how-to-install-nginx-on-ubuntu-18-04) the best, which have different pages per version).
> 
> We'll have to make a call as to what versions to officially support and test on later. I did a quick search, but couldn't find a "Ubuntu server market share by version" report. If anyone has a pointer or more insight, please share.

Generally speaking, the top 3 Linux Distros (LTS versions) are Ubuntu, Debian, and RHEL/Centos though given the recent change to Centos that will probably change to RockyLinux later this year.

My advice is stick to LTS versions, for Ubuntu this happens every 2 years (any of the .04 releases), though the support terms for older versions do last a few years, the bulk of the packages won't be upgraded to major releases only bug fixes. For example Nginx in the default Ubuntu PPAs is 8 versions behind (Xenial vs Focal): https://launchpad.net/ubuntu/+source/nginx

I'll try and work on a 20.04 page this week as the differences aren't all that major (I got a mattermost running on 20.04 in about 10 minutes).

**Comment by justinegeffen** [comment]
The documentation is now available here: https://docs.mattermost.com/guides/boards.html. There is no installation content in these docs and the Personal Edition content isn't included in those docs - you can find them here: https://github.com/mattermost/focalboard/tree/main/website/site/content/download/personal-edition for now. :)
